modules/p2/ding_list/include/form.inc

```diff
@@ -347,13 +347,7 @@ function ding_list_create_list_form_submit($form, &$form_state) {
       return;
     }
 
-    drupal_set_message(t(
-        "You've create the list !list.<br/>Read more about how you use the lists !help",
-        array(
-          '!list' => l($list->title, 'list/' . $list->id),
-          '!help' => l(t('Here'), 'help/lists'),
-        )
-      ));
+    drupal_set_message(theme('ding_list_create_info', array('list' => $list)));
```

Comment:
It is preferable to replace direct calls to the theme() function with calls to drupal_render() by passing a render array with a #theme key to drupal_render(), which in turn calls theme().